.gitignore

```
@@ -27,4 +27,4 @@ qchem/tests/tmp/*
 qchem/tests/tmp/timer.dat
 timer.dat
 tmp/*
-benchmark/*
+benchmark/revisions/
```

Was curious about these warnings coming from GitHub, found the following two answers useful:
* https://stackoverflow.com/a/5813359/12899253
* https://stackoverflow.com/a/729795/12899253

```suggestion
benchmark/revisions/

```